### ISSUE TITLE
Data cart shows always empty

### ISSUE THREAD

**Issue by atverm** [ICOS-Carbon-Portal/data#256]
Data cart should show when there are data objects in it, now the data cart always looks empty

**Comment by mirzov** [comment]
I cannot reproduce the problem, wither with Firefox or with Chromium. Are you logged in? Which browser did you use?

**Comment by atverm** [comment]
Chrome on Windows

***@***.***


***@***.***

**Comment by jonathanthiry** [comment]
Do I understand correctly that you are talking about the data cart link not showing when there is data in the cart?

![Screenshot 2022-05-20 at 13-43-54 Carbon Portal's data search](https://user-images.githubusercontent.com/991720/169521680-287c28a8-3e0f-4528-a82f-d9c4e4c6d7a6.png)

**Comment by eee-ea** [comment]
Yes, there should be something in the cart, it should change colour and/or items, maybe even the number of items

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by jonathanthiry** [comment]
Can you confirm that there is something in the cart after adding data to it (https://data.icos-cp.eu/portal/#{%22route%22:%22cart%22}) and that the issue is only about the link not giving enough information, like I showed on the previous screenshot?

**Comment by atverm** [comment]
Btw that data cart icon in the header is never visible on a phone screen



Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by eee-ea** [comment]
Aha, it is about the icon in the page header, the physically visible data cart

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by jonathanthiry** [comment]
Thanks for clarification!

You have to tap on the menu button to access the cart on small screens:

![Screen Shot 2022-05-20 at 14 02 24](https://user-images.githubusercontent.com/991720/169525010-4178a0cb-bf31-4242-9711-ddf0e64bdc0a.png)
